custom resource returned: Error: 'agentVersion'

### Describe the bug

custom resource returned error when creating Agent (errorred out in us-west-2, us-east-1 seems to be fine):

[BedrockCRProviderCustomResources log:
[ERROR] KeyError: 'agentVersion'
Traceback (most recent call last):
  File "/var/task/custom_resources/__init__.py", line 47, in on_event
    return on_event_bedrock_agent_alias(event, context)
  File "/var/task/custom_resources/bedrock_agent_alias.py", line 81, in on_event
    return on_create(event, str(uuid.uuid1()))
  File "/var/task/custom_resources/bedrock_agent_alias.py", line 311, in on_create
    response = create_agent_alias(agent_id, alias_name, session, agent_version, client_token)
  File "/var/task/tenacity/__init__.py", line 289, in wrapped_f
    return self(f, *args, **kw)
  File "/var/task/tenacity/__init__.py", line 379, in __call__
    do = self.iter(retry_state=retry_state)
  File "/var/task/tenacity/__init__.py", line 314, in iter
    return fut.result()
  File "/var/lang/lib/python3.12/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
  File "/var/lang/lib/python3.12/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/var/task/tenacity/__init__.py", line 382, in __call__
    result = fn(*args, **kwargs)
  File "/var/task/custom_resources/bedrock_agent_alias.py", line 200, in create_agent_alias
    agent_version = get_version(
  File "/var/task/custom_resources/bedrock_agent_alias.py", line 169, in get_version
    return routing_configuration[0]["agentVersion"]

### Expected Behavior

Agent create success

### Current Behavior

cdk stack deployment fails

### Reproduction Steps

import * as cdk from 'aws-cdk-lib';
import { bedrock } from '@cdklabs/generative-ai-cdk-constructs';
import { Construct } from 'constructs';

export interface KbStatefulProps extends cdk.StackProps {
}

export class KbStatefulStack extends cdk.Stack {

  constructor(scope: Construct, id: string, props: KbStatefulProps) {
    super(scope, id, props);

    const knowledgeBase = new bedrock.KnowledgeBase(this, 'BedrockKnowledgeBase', {
      embeddingsModel: bedrock.BedrockFoundationModel.TITAN_EMBED_TEXT_V1,
      instruction: `you are a knowledgeable base that answers all questions`,
    });

    const tamAgent = new bedrock.Agent(this, 'MyAgent', {
      name: 'MyAgent',
      description: 'The agent for consultation on any example issues.',
      foundationModel: bedrock.BedrockFoundationModel.ANTHROPIC_CLAUDE_V2,
      instruction:
        'You are a useful agent that provides details or advices related to any issues.',
      idleSessionTTL: cdk.Duration.minutes(15),
      knowledgeBases: [knowledgeBase],
      shouldPrepareAgent: false,
      aliasName: 'exampleAliasName',
    });

  }
}

### Possible Solution

bedrock_agent_alias.py line 169, change to 'return routing_configuration[0].get("agentVersion")'

### Additional Information/Context

_No response_

### CDK CLI Version

2.139.0 (build 2699ffd)

### Framework Version

0.1.134

### Node.js Version

20

### OS

Linux

### Language

Typescript

### Language Version

_No response_

### Region experiencing the issue

us-west-2

### Code modification

no

### Other information

_No response_

### Service quota

- [X] I have reviewed the service quotas for this construct

--- a/lambda/bedrock-custom-resources/custom_resources/bedrock_agent_alias.py
+++ b/lambda/bedrock-custom-resources/custom_resources/bedrock_agent_alias.py
@@ -166,7 +166,10 @@ def list_unused_agent_versions(agent_id: str, session: boto3.Session) -> List[in
 
 def get_version(routing_configuration: List[Dict]) -> str:
     if len(routing_configuration) == 1:
-        return routing_configuration[0]["agentVersion"]
+        try:
+            return routing_configuration[0]["agentVersion"]
+        except KeyError:
+            return None
     else:
         return None
 

